android/java/org/chromium/chrome/browser/rewards/BraveRewardsPanel.java

```diff
@@ -111,7 +111,6 @@ public class BraveRewardsPanel
     private static final String SUPPORT_URL = "https://support.brave.com";
     private static final String BRAVE_REWARDS_PAGE = "https://brave.com/rewards";
     private static final String BRAVE_REWARDS_CHANGES_PAGE = "https://brave.com/rewards-changes";
-    private static final String BRAVE_REWARDS_RESET_PAGE = "brave://rewards#reset";
```

Comment:
Moved `BRAVE_REWARDS_RESET_PAGE` to `BraveActivity` to make it accessible from other code.